gloo/docs/user_guides/gateway/configuring_route_options/advanced_routing_action.md

```diff
@@ -128,7 +136,7 @@ Second, you need to add a [`subset`]({{% ref "/v1/github.com/solo-io/gloo/projec
 within the [`Destination` spec]({{% ref "/v1/github.com/solo-io/gloo/projects/gloo/api/v1/proxy.proto.sk#destination" %}})
 of the Route Action.
 
-Following is an example of using a label, e.g., `color: blue`, to subset pods handling requests.
+Following is an example of using a label, e.g. `color: blue`, to subset pods handling requests.
```

Comment:
When reading this I was confused when it jumped to an Upstream manifest from a doc page on how to write Virtual Service manifests. I feed it would help if the lead into this example stated something like, "The Upstream allows you to subset all requests to that upstream as you see in the following example. Within the routeAction you can also define a similar subset that is applied only for a given route".

It feels like a little more is needed to help the user understand that they are still in the Virtual Service Routing documentation and you're referencing an Upstream as an example

Reply:
Done

Reply:
I agree, the subset configuration is not easy to understand, particularly why you need to add config in two places. I will add some more detail and a link to the [Envoy docs](https://github.com/envoyproxy/envoy/blob/master/source/docs/subset_load_balancer.md) that explain how subset load balancing works.